doc/website/_get-started/01-overview.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Overview"
+permalink: /get-started/overview/
+excerpt: "Overview of ROHD framework."
+last_modified_at: 2022-12-05T21:36:18-04:00
```

Comment:
Are these files generated by jekyll or did you copy paste them all throughout into different files and hierarchy?

Reply:
Fixed and removed unnecessary flag and options.